### DISCUSSION TITLE
hello tmijail

### DISCUSSION THREAD

**Discussion by iqbalnurhabib** [tmijail/Odoo-Power-BI-Connector#23] [Q&A] [answered]
![image](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/49646304/20ef52d7-fa97-42df-a184-c9f436e7d23b)

database can't connect, looks like an error occurred in res.user.parent_id?

**Comment by tmijail** [comment, marked as answer]
This is fixed in the new releases [v1.0.0](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/v1.0.0) and [v0.2.0](https://github.com/tmijail/Odoo-Power-BI-Connector/releases/tag/v0.2.0).

**Comment by iqbalnurhabib** [comment]
Thank you very much
